erpnext/buying/doctype/supplier/supplier.py

```
@@ -50,7 +50,7 @@ class Supplier(TransactionBase):
 		disabled: DF.Check
 		email_id: DF.ReadOnly | None
 		gender: DF.Link | None
-		hold_type: DF.Literal["", "All", "Invoices", "Payments"]
+		hold_type: DF.Literal["All", "Invoices", "Payments"]
```

Ruff (0.15.14)

[error] Undefined name `All`

(F821)